docs/routing.md

```diff
@@ -91,6 +91,22 @@ export default {
 }
 ```
 
+If it's a nested route, paths that start with `/` will be treated as a root path:
+
+```js
+// pages/information/about.vue
+
+export default {
+  nuxtI18n: {
+    paths: {
+      en: '/about-us', // -> accessible at /about-us (no prefix since it's the default locale)
+      fr: 'a-propos', // -> accessible at /fr/information/a-propos
```

Comment:
I've been testing this change and the page is actually accessible at `/fra-propos`.
In this case, the expected config would be to specify the full path for each locale, ie:

```js
paths: {
  en: '/about-us',             // -> accessible at /about-us (no prefix since it's the default locale)
  fr: '/information/a-propos', // -> accessible at /fr/information/a-propos
  es: '/sobre'                 // -> accessible at /es/sobre
}
```

If we wanted to handle the case without the leading `/`, it would require more checks in the routes generation, which would lead to additional complexity in setting up a project with nuxt-i18n. I'm not sure we need to handle this for now.

Reply:
Sorry, I did the condition below incorrectly earlier. With the latest commit, we will add prefix when
1. it is top level path
2. the path starts with `/`

No complexity is added since vue-router already supports this path pattern. https://router.vuejs.org/guide/essentials/nested-routes.html here is says `Note that nested paths that start with / will be treated as a root path. This allows you to leverage the component nesting without having to use a nested URL.`

`path.startsWith('/')` is added to match with vue-router's design. This example is for nested routes only. Non-nested routes should always start with `/`.